api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -192,6 +192,10 @@ type AgentClusterInstallSpec struct {
 	// +optional
 	PlatformType PlatformType `json:"platformType,omitempty"`
 
+	// ExternalSpec is the spec for the external platform
+	// +optional
+	ExternalSpec ExternalSpec `json:"external,omitempty"`
```

Comment:
```suggestion
	ExternalSpec* ExternalSpec `json:"external,omitempty"`
```
I think it needs to be a pointer to work along with "omitempty"

Reply:
```suggestion
	ExternalSpec *ExternalSpec `json:"external,omitempty"`
```

Reply:
No need to guess - please just copy-paste
https://github.com/openshift/api/blob/edd317298780d562af7d5d979b237d14fc299785/config/v1/types_infrastructure.go#L311-L314